### PR TITLE
Fix: Change scraping logic for ICRC website

### PR DESCRIPTION
**Pull request by susilnem**
Addresses
- https://github.com/IFRCGo/go-api/issues/2287

## Changes
- This PR is related to the command `api/management/commands/ingest_icrc.py`
-  Change the ingestion logic
- Need to run the command `python manage.py ingest_icrc`

## Checklist
Things that should succeed before merging.

- [ ] Updated/ran unit tests
- [ ] Updated CHANGELOG.md

## Release

If there is a version update, make sure to tag the repository with the latest version.

### CODE REVIEW

File: api/management/commands/ingest_icrc.py [hunk 1]
```diff
@@ -36,32 +36,36 @@ def handle(self, *args, **kwargs):
...
-        # Get the countries information from the "Where we work" page
-        regions_list = soup.find("div", {"id": "blockRegionalList"}).find_all("ul", {"class": "list"})
+        # Get countries information from "Where we work" page
+        countries = soup.find("div", {"class": "js-select-country-list"}).find("ul").find_all("ul")
+
         country_list = []
-        for region in regions_list:
-            for country in region.find_all("li", {"class": "item"}):
-                # Get key information
+        for region in countries:
```
Comment (Rup-Narayan-Rajbanshi) [inline api/management/commands/ingest_icrc.py:43]: can we write this to `for region in regions_and_countries` or `for region in regions`  ? or give some meaningful name.

**Review by Rup-Narayan-Rajbanshi — APPROVED** [review]

**Review by szabozoltan69 — APPROVED** [review]

**Comment by szabozoltan69** [comment]
Should we merge this?

**Comment by szabozoltan69** [comment]
Deployed to Staging

**Comment by susilnem** [comment]
hey @szabozoltan69 
we also need to run this command `python manage.py ingest_icrc`

**Comment by szabozoltan69** [comment]
Done:
```
~/go-api# python manage.py ingest_icrc
2024-11-04 15:00:22 INFO     Starting ICRC data ingest
2024-11-04 15:01:10 INFO     77 ICRC added
2024-11-04 15:01:10 WARNING  instance does not have an es_id() method
```

**Pull request merged**